analyzer-comments/elixir/high-school-sweetheart/multiline_string.md

```diff
@@ -0,0 +1,29 @@
+# multiline string
+
+To comfortably work with texts with a lot of newlines, use the triple-double-quote heredoc syntax.
+
+```elixir
+# preferred
+
+"""
+     ******       ******
+   **      **   **      **
+ **         ** **         **
+**            *            **
+**                         **
+**     #{i1}  +  #{i2}     **
+ **                       **
+   **                   **
+     **               **
+       **           **
+         **       **
+           **   **
+             ***
+              *
+"""
+
+# not preferred
+
+"     ******       ******\n   **      **   **      **\n **         ** **         **\n**            *            **\n**                         **\n**     #{i1}  +  #{i2}     **\n **                       **\n   **                   **\n     **               **\n       **           **\n         **       **\n           **   **\n             ***\n              *\n"
```

Comment:
Haha, this will most likely break the interface a bit, but I guess that's the point of this check to begin with :)